server/services/source-map.js

```diff
@@ -7,20 +7,49 @@ const libSourceMaps = istanbulAPI.libSourceMaps;
 
 const sourceMap = libSourceMaps.createSourceMapStore({verbose: Conf.IS_COVERAGE_ACTIVE});
 const meteor_dir = Conf.COVERAGE_APP_FOLDER;
-const regexAlterationSourceMapPath = new RegExp(/(packages\/)([a-zA-Z-]*)[_:]([a-zA-Z-_]*)(.*)/);
+// Next RegExps must consider following paths:
+//  - meteor://💻app/packages/pkgAuthor[_:]pkgName/...
+//  - meteor://💻app/packages/local-test:pkgAuthor:pkgName/...
+const regexAlterationSourceMapPath = new RegExp(/(packages\/)(?:local-test[_:])?([a-zA-Z-]*)[_:]([a-zA-Z-_]*)(.*)/);
+const regexPUT = new RegExp(/packages\/local-test:([a-zA-Z-_]*):([a-zA-Z-_]*)\/.*/);
 
 // Alter inside the source map the path of each sources
 alterSourceMapPaths = function (map) {
   var match;
+
+  if (!Meteor.PUT && Meteor.isPackageTest && meteor_dir === `${process.env.PWD}/`) {
```

Comment:
Instead of using `Meteor.PUT`, you can create a new function `SourceMap.extractPackageInfo` that returns undefined or the object `{author, name}`. But yes it's the good place to do that.